rts/Lua/LuaUnsyncedRead.cpp

```diff
@@ -2725,14 +2725,23 @@ int LuaUnsyncedRead::GetCameraNames(lua_State* L)
 
 /***
  * @function Spring.GetCameraState
- * @param useReturns false
- * @return CameraState cameraState
- */
-/***
- * @function Spring.GetCameraState
- * @param useReturns true? (Default: `true`) Return multiple values instead of a table.
- * @return CameraName name
- * @return any ... depend on current controller mode.
+ * @param useReturns [boolean] (optional, default: true)
+ *   If true or omitted, returns a table with all camera state fields. If false, returns the camera name and a variable number of camera-specific values as separate return values.
+ *
+ * @return table cameraState (if useReturns is true or omitted)
```

Comment:
This return was correctly documented already, returning an instance of [`CameraState`](https://beyond-all-reason.github.io/RecoilEngine/docs/lua-api/#CameraState)

Reply:
You are right! Indeed it seems the true and false values as well as the description of the param must be fixed (switch true and false and fix the description of `useReturns` (maybe name it `useTable`?).

Thank you for reporting and finding it.

But we'd still like to keep the overload format.